extensions/SemanticForms/includes/SF_ParserFunctions.php

```diff
@@ -212,7 +212,7 @@ static function renderQueryFormLink ( &$parser ) {
 		return $parser->insertStripItem( self::createFormLink( $parser, $params, 'queryformlink' ), $parser->mStripState );
 	}
 
-	static function renderFormInput( &$parser ) {
+	static function renderFormInput( $parser ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This function "renderFormInput" has 175 lines, which is greater than the 150 lines authorized. Split it into smaller functions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS138)